Add content from community doc PR #728 (#64)
rancher/harvester-product-docs: versions/v1.5/modules/en/pages/virtual-machines/backup-restore.adoc

[NOTE]
====

> A backup target must be set up. For more information, see <<Configure Backup Target>>. If the backup target has not been set, you'll be prompted with a message to do so.

Reviewer: ```suggestion
A backup target must be configured. For more information, see <<Configure Backup Target>>. If no backup target is set, a message prompts you to configure one.
```
